src/components/ScreenreaderText/ScreenreaderText.tsx

```diff
@@ -3,10 +3,23 @@ import '../../yamui';
 import * as React from 'react';
 import { join } from '../../util/classNames';
 import { NestableBaseComponentProps as ScreenreaderTextProps } from '../../util/BaseComponent/props';
-import './ScreenreaderText.css';
 
 export { ScreenreaderTextProps };
 
+/* These styles are set inline to reduce the liklihood of being overridden. Similar to !important */
+const inlineStyles: React.CSSProperties = {
```

Comment:
i would check with @BrendanMcK here. why not just keep `!important` to know we won't be changing any behavior, and use getStyles for consistency?

Reply:
I assumed that merge-styles didn't allow `!important`, but I think I'm probably wrong, since it seems to allow you to set strings for everything.  I'll revisit this in a bit.

Reply:
`clip`, `position` and `overflow` will not accept a string type, so I don't know how to set `!important` on those.  @swese44 any ideas?

Reply:
How is `!important` needed in these places? Was it to protect from a dev overriding these properties through CSS? But that's not even going to be possible now, right?

Reply:
Yeah, I don't think it should be necessary here anymore.